src/mongocxx/search_index_view.hpp

```diff
@@ -0,0 +1,122 @@
+#pragma once
+
+#include <string>
+#include <vector>
+
+#include <bsoncxx/document/value.hpp>
+#include <bsoncxx/document/view_or_value.hpp>
+#include <bsoncxx/stdx/optional.hpp>
+#include <bsoncxx/string/to_string.hpp>
+#include <mongocxx/client_session.hpp>
+#include <mongocxx/cursor.hpp>
+#include <mongocxx/options/search_index_view.hpp>
+#include <mongocxx/search_index_model.hpp>
+
+#include <mongocxx/config/prelude.hpp>
+
+namespace mongocxx {
+MONGOCXX_INLINE_NAMESPACE_BEGIN
+
+///
+/// Class representing a MongoDB search index view.
+///
+class MONGOCXX_API search_index_view {
+   public:
+    search_index_view(search_index_view&&) noexcept;
+    search_index_view& operator=(search_index_view&&) noexcept;
+
+    ~search_index_view();
+
+    ///
+    /// @{
+    ///
+    /// Returns a cursor over all the search indexes.
+    ///
+    cursor list(const bsoncxx::document::view& aggregation_opts = bsoncxx::document::view{});
+
+    cursor list(const std::string name,
+                const bsoncxx::document::view& aggregation_opts = bsoncxx::document::view{});
```

Comment:
```suggestion
                const bsoncxx::document::view& aggregation_opts = bsoncxx::document::view{});
                
    ///
    /// @}
    ///
```

Comment to close the doxygen group.

Doxygen docs can be built and viewed locally:
```
mkdir -p ./build/docs/api/current  
cmake --build cmake-build --target doxygen-current
open ./build/docs/api/current/index.html  
```